src/tests/ftest/launch.py

```diff
@@ -2225,12 +2357,14 @@ def _setup_test_directory(self, test):
         """
         logger.debug("-" * 80)
         test_dir = os.environ["DAOS_TEST_LOG_DIR"]
+        user_dir = os.environ["DAOS_TEST_USER_DIR"]
         logger.debug("Setting up '%s' on %s:", test_dir, test.host_info.all_hosts)
         commands = [
             f"sudo -n rm -fr {test_dir}",
             f"mkdir -p {test_dir}",
             f"chmod a+wr {test_dir}",
             f"ls -al {test_dir}",
+            f"mkdir -p {user_dir}"
```

Comment:
Is this needed?

Reply:
Yes, because someone **could** override the path to be anywhere else:
https://github.com/daos-stack/daos/blob/8b4fd08e41d1848a2088aba6b43811412447069a/src/tests/ftest/launch.py#L1155-L1156

And `useradd` doesn't create parent directories.

Reply:
Ah, I misread this as a duplicate of the mkdir above but I see it's two different directories now.